app/networkapi/settings.py

```diff
@@ -45,7 +46,9 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = env('DEBUG')
 
-ALLOWED_HOSTS = env('ALLOWED_HOSTS')
+ALLOWED_HOSTS = os.getenv("ALLOWED_HOSTS").split(",")
```

Comment:
if you're using getenv, you can make this safer as `ALLOWED_HOSTS = os.getenv("ALLOWED_HOSTS", '').split(",")` so that the fallback value is guaranteed safe